app/src/public/css/login-publishing.css

```
@@ -0,0 +1,168 @@
+* {
+    box-sizing: border;
+    color: #c4c4c4;
+    font-family: "Noto Sans KR", sans-serif;
+}
+.wholeWrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* background-color: rgb(226, 226, 226); */
+    width: 100vw;
+    height: 100vh;
+}
+.circleImg {
+    margin: 110px 0 50px 0;
+    width: 230.15px;
+    height: 100px;
+    display: block;
+    background-image: url("../images/동그라미.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center center;
+}
+.mainWrapper {
+    margin: 0 0 0 0;
+    width: 780px;
+    height: 650px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 2px solid #90e2d6;
+    border-radius: 16px;
+}
+.title {
+    font-size: 50px;
+    color: black;
+    margin: 46px 0 38px;
+    font-family: "Noto Sans KR", sans-serif;
+}
+.inputBox {
+    display: flex;
+    flex-direction: column;
+    /* justify-content: center; */
+    align-items: center;
+}
+.inputInfo {
+    width: 100%;
+    height: 65px;
+    border: none;
+    font-size: 18px;
+    border: none;
+    outline: none;
+}
+.inputEach {
+    display: flex;
+    align-items: center;
+    width: 505px;
+    margin: 0 0 25px;
+    border-bottom: solid 1px #c4c4c4;
+}
```

이거 아마도 각 input태그에 대한 값이겠죠?
클래스 안주고 이렇게 해도 됩니다
```suggestion
.inputBox input {
    display: flex;
    align-items: center;
    width: 505px;
    margin: 0 0 25px;
    border-bottom: solid 1px #c4c4c4;
}
```

---

조금 다듬겠습니다👍👍